core/src/scss/components/date-stacked/_date-stacked.scss

```diff
@@ -15,13 +15,22 @@
   @include modular-spacing('padding', 0);
   background: $color-black;
   color: $color-white;
+  max-width: 87px;
   text-align: center;
 
-  .su-date-month {
+  @include grid-media-only('md') {
+    max-width: 101px;
```

Comment:
Before you commit to this convention please do some research on the right approach on bounding widths in components. I'd like to know what others are doing and why.

Reply:
Walmart: they are hardcoding width to many components, including cards, tiles, and buttons.

Bootstrap: Unless something is assigned a width of 100%, it's more rare that the framework hardcodes component widths. In the cases where it does (e.g., popover and tooltips) it has a corresponding variable (e.g., `$popover-max-width` and `$tooltip-arrow-width`).

Lonely Planet: they are hardcoding widths, and are even naming some components in direct relation to their width (e.g., "Double width card with cover image").

WVU: most components are inherently 100% width. They do hard code width of cards when inside of a component-like container.

U.S. Web Design System: most components are inherently 100% width. In the cases where it does assign a width, it has a corresponding variable.

IBM Carbon: assigns width to small (atom-level components).

Reply:
There is certainly a mix of what others are doing. I'm in the camp of the middle-of-the-road approach, which I see as being: we can assign widths to smaller (atom-level components like the date stacked component), but pass the width through with a variable that can be more easily changed/overridden.

Reply:
Great, that sounds sensible to me. I have created you a ticket to document this decision and linked it to this thread for a later time.

Reply:
I'm fine with this max-width treatment.